examples/flutter_gallery/lib/welcome/steps/documentation_step.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import '../constants.dart';
+import '../welcome_step_state.dart';
+import 'step_container.dart';
+
+const String _kTitle = 'Complete, flexible APIs';
+const String _kSubtitle = 'View full API documentation, when you need it, with a quick tap. Look for the documentation icon in the app bar.';
+
+class DocumentationWelcomeStep extends StatefulWidget {
+  const DocumentationWelcomeStep({Key key}) : super(key: key);
+  @override
+  DocumentationWelcomeStepState createState() => DocumentationWelcomeStepState();
```

Comment:
If you create an interface, you can make all the states private, which is cleaner.